cmd/kubeadm/app/phases/bootstraptoken/node/token.go

```diff
@@ -42,10 +41,7 @@ func UpdateOrCreateTokens(client clientset.Interface, failIfExists bool, tokens
 
 		secretName := bootstraputil.BootstrapTokenSecretName(token.Token.ID)
 		secret, err := client.CoreV1().Secrets(metav1.NamespaceSystem).Get(context.TODO(), secretName, metav1.GetOptions{})
-		if err != nil && !apierrors.IsNotFound(err) {
-			return err
-		}
-		if secret != nil && failIfExists {
+		if secret != nil && err == nil && failIfExists {
```

Comment:
Maybe the code I wrote before is correct.
```go
else if err != nil && !apierrors.IsNotFound(err) {
			return err
}
```

Reply:
If you are not available, I will test this problem as soon as possible. There are some travel plans recently 😀.

Reply:
we should revert the change to unblock the ci first.

https://github.com/kubernetes/kubernetes/pull/115068#issuecomment-1382456271

looking for lgtm.

Reply:
> we should revert the change to unblock the ci first.
> 
> [#115068 (comment)](https://github.com/kubernetes/kubernetes/pull/115068#issuecomment-1382456271)
> 
> looking for lgtm.

got it